geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/CacheClientProxyStats.java

```diff
@@ -60,6 +60,8 @@
   private static final String DELTA_FULL_MESSAGES_SENT = "deltaFullMessagesSent";
   /** Name of the CQ count statistic */
   private static final String CQ_COUNT = "cqCount";
+  /** Name of the number of threads currently adding message in queue statistic */
+  private static final String NUMBER_THREADS_PUT_IN_QUEUE = "numberThreadsPutInQueue";
```

Comment:
"number" is not needed in the name since all stats are numeric. Given that we have a stat named "messagesQueued", I think this one should be named "messagesWaitingToQueue".